builder/assets/camera_info/main_camera.yaml

```diff
@@ -1,7 +1,7 @@
 image_width: 320
 image_height: 240
 distortion_model: plumb_bob
-camera_name: raspicam
+camera_name: main_camera
```

Comment:
Just in case, `camera_info_manager` will attempt to load `.ros/camera_info/main_camera_optical.yaml` due to cv_camera using `frame_id` as the camera name: https://github.com/OTL/cv_camera/blob/22973af7f549298801bd258e73f3a21a66974118/src/capture.cpp#L19

Reply:
Hm, that’s wierd.

Reply:
I believe we could still get around that without setting the camera name to something weird like "main_camera_optical". `camera_info_manager` supports (very limited) variable substitutions in camera URL (https://github.com/ros-perception/image_common/blob/hydro-devel/camera_info_manager/src/camera_info_manager.cpp#L333), so we could technically set the URL to `${ROS_HOME}/camera_info/main_camera.yaml` and still enjoy `set_camera_info` overwriting the calibration data.

Reply:
I believe that’s better to fix cv_camera to make it load calibration by node’s name.  ?

Reply:
Yes, that would be the best outcome. I've sent them a pull request: https://github.com/OTL/cv_camera/pull/26